### PR TITLE
feat: update total miles and time

### PR DESCRIPTION
**Pull request by marco-esposito**

### CODE REVIEW

File: consumer.js [hunk 1]
```diff
@@ -34,7 +35,19 @@ class QueueConsumer extends Consumer {
...
+    const updateTotalTimeMiles = async () => {
+      const trips = await Trip.find({mac_address: message.mac_address});
+      if (trips.length) {
+        const lastTrip = trips.slice(-1)[0];
+        const distance = lastTrip.distance;
+        const time = lastTrip.end_time - lastTrip.start_time;
+        const company = await Company.findOne({fleet: {$elemMatch: {mac_address: message.mac_address}}});
+        const vehicles = company.fleet.filter(vehicle => vehicle.mac_address === message.mac_address);
+        vehicles[0].total_driving_time += time;
+        vehicles[0].total_miles_driven += distance;
```
Comment (arol) [inline consumer.js]: Distance is evaluated in KMs.

File: index.js [hunk 2]
```diff
@@ -16,9 +22,8 @@ app
 
 app.listen(process.env.PORT || 3000);
 
-const config = require('./smqConfig');
-const monitorServer = require('redis-smq').monitor(config);
+// const monitorServer = require('redis-smq').monitor(config);
```
Comment (arol) [inline index.js]: Remove commented code.

File: smqConfig.js [hunk 3]
```diff
@@ -13,9 +13,9 @@ module.exports = {
       level: 'debug',
     },
   },
-  monitor: {
-    enabled: true,
-    host: '127.0.0.1',
-    port: 3001,
-  },
+  // monitor: {
```
Comment (arol) [inline smqConfig.js]: Remove commented code.

**Review by arol — CHANGES_REQUESTED** [review]

**Review by marco-esposito — COMMENTED** [review]
- Moved from Miles to Km
- Comments removed

**Review by arol — APPROVED** [review]

**Pull request closed without merging**